i18n/ch.yaml

```diff
@@ -0,0 +1,30 @@
+- id: blogTitle
+  translation: Rotations Récentes
+- id: readMore
+  translation: Lire la suite
+- id: viewAllPost
+  translation: Voir tous les articles
+- id: share
+  translation: Partager
+- id: aboutAuthor
+  translation: À propos de l'auteur
+- id: authorsPosts
+  translation: Articles de l'auteur
+- id: all
+  translation: All
+- id: about
+  translation: À propos
+- id: services
+  translation: Services
+- id: portfolio
+  translation: Portfolio
+- id: team
+  translation: Équipe
+- id: pricing
+  translation: Tarification
+- id: blog
+  translation: Blog
+- id: contact
+  translation: Contact
+- id: themefisher
+  translation: Themefisher
```

Comment:
We can remove this file (we'll add Chinese-language in a future PR using the go-18n for Hugo approach)